04-using-github-with-the-platform.Rmd

```diff
@@ -196,19 +196,23 @@ The platform has configured simple "safety barriers" to reduce risk of accidenta
 
 ### Private and Public Repositories
 
-The majority of repos in the DASD GitHub organisation are **Private**: this keeps visibility, access and editing rights controlled to particular groups within the MoJ network and is sensible in most cases. Owners and Admins of a repository can control this visibility by going to **_Settings_ > _Collaborators & Teams_** and adding teams to read/write/admin access groups. To maximise collaboration _within_ the organsiation but still keep code that shouldn't be in the public domain (e.g. unpublished commentary on a `.Rmd` file) hidden from anyone _outside_ the organisation, you can add the group `everyone` here. This also makes your code [searchable.](https://moj-analytical-services.github.io/platform_user_guidance/using-github-with-the-platform.html#other-tips-and-tricks-work-in-progress)
+GitHub repositories can be public or private:
 
-### Private R Packages for Reproducible Analysis
+*Public repos* means visible to the world. A repo should contain no passwords/secrets and no data (apart from small reference tables) - these are particulary important for public repos.
```

Comment:
I think this bit about
>"A repo should contain no passwords/secrets and no data (apart from small reference tables)"

ought to be above the *Public repos* section, immediately following the statement that Github repos can be public or private. We can then add the particular importance of this for public repos here to really drive it home.